test/cmdlineTests/evm_version_constantinople/err

```diff
@@ -0,0 +1,4 @@
+Warning: Support for constantinople and older EVM versions is deprecated and will be removed in the future.
+
```

Comment:
Without looking through the PR completely so far, it's a bit disconcerting that the newline here is apparently contingent on whether it's a release build or not (or where is https://app.circleci.com/pipelines/github/ethereum/solidity/31397/workflows/2727648f-a34a-4ea7-a27e-72eb0b4c2470/jobs/1398946 coming from?)

Reply:
It's a leftover from the pre-release warning being stripped. Looks like the warning has an empty line after it and we're not stripping it. It went unnoticed probably because stripping of leading whitespace from the output used to remove it anyway. Now, with another warning before it, it no longer happens.

I tweaked the regexes to remove the empty line.